ddtrace/profiling/exporter/http.py

```diff
@@ -80,6 +80,9 @@ def __attrs_post_init__(self):
                 "profiler_version": ddtrace.__version__.encode("ascii"),
             }
         )
+        lambda_function_name = os.environ.get("AWS_LAMBDA_FUNCTION_NAME")
+        if lambda_function_name:
+            tags.update({"functionname": lambda_function_name.encode()})
```

Comment:
I'd probably move that in the `Profiler` class so it can be used by any other exporter in the future.